apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/AppTab.tsx

```diff
@@ -1,27 +1,166 @@
+    },
+  ];
+
+  const waitTime = () => {
+    if (survey.recontactDays !== null) {
+      let waitingTime = "";
+
+      if (survey.recontactDays === 0) {
+        waitingTime = t("environments.surveys.summary.in_app.display_criteria.time_based_always");
+      } else if (survey.recontactDays === 1) {
+        waitingTime = `${survey.recontactDays} ${t("environments.surveys.summary.in_app.display_criteria.time_based_day")}`;
+      } else {
+        waitingTime = `${survey.recontactDays} ${t("environments.surveys.summary.in_app.display_criteria.time_based_days")}`;
+      }
+      return `${waitingTime} ${t("environments.surveys.summary.in_app.display_criteria.overwritten")}`;
+    }
+    if (project.recontactDays !== null) {
+      if (project.recontactDays === 0) {
+        return `${project.recontactDays} ${t("environments.surveys.summary.in_app.display_criteria.time_based_always")}`;
+      } else if (project.recontactDays === 1) {
+        return `${project.recontactDays} ${t("environments.surveys.summary.in_app.display_criteria.time_based_day")}`;
+      } else {
+        return `${project.recontactDays} ${t("environments.surveys.summary.in_app.display_criteria.time_based_days")}`;
+      }
+    }
+    return t("environments.surveys.summary.in_app.display_criteria.time_based_always");
+  };
+
+  const displayOption = () => {
+    if (survey.displayOption === "displayOnce") {
+      return t("environments.surveys.edit.show_only_once");
+    } else if (survey.displayOption === "displayMultiple") {
+      return t("environments.surveys.edit.until_they_submit_a_response");
+    } else if (survey.displayOption === "respondMultiple") {
+      return t("environments.surveys.edit.keep_showing_while_conditions_match");
+    } else if (survey.displayOption === "displaySome") {
+      return t("environments.surveys.edit.show_multiple_times");
+    }
+  };
 
   return (
-    <div className="flex h-full grow flex-col">
-      <OptionsSwitch
-        options={[
-          { value: "webapp", label: t("environments.surveys.summary.web_app") },
-          { value: "mobile", label: t("environments.surveys.summary.mobile_app") },
-        ]}
-        currentOption={selectedTab}
-        handleOptionChange={(value) => setSelectedTab(value)}
-      />
+    <div className="flex flex-col justify-between space-y-6 pb-4">
+      <div className="flex flex-col space-y-6">
+        <Alert variant={environment.appSetupCompleted ? "success" : "warning"} size="default">
+          <AlertTitle>{t("environments.surveys.summary.in_app.connection_title")}</AlertTitle>
+          <AlertDescription>
+            {t("environments.surveys.summary.in_app.connection_description")}
+          </AlertDescription>
+          {!environment.appSetupCompleted && (
+            <AlertButton asChild>
+              <Link href={`/environments/${environment.id}/project/app-connection`}>
+                {t("environments.surveys.summary.in_app.connection_button")}
```

Comment:
```suggestion
                {t("common.connect_formbricks")}
```

We already have a fitting Key val pair for the button